Sources/Atoms/Context/AtomContext.swift

```diff
@@ -102,7 +102,26 @@ public protocol AtomContext {
     /// ```
     ///
     /// - Parameter atom: An atom to reset.
-    func reset(_ atom: some Atom)
+    func reset<Node: Atom>(_ atom: Node)
```

Comment:
It needs `@_disfavoredOverload` attribute as it's ambiguous overload.

Reply:
Good catch, fixed!